src/_includes/layouts/base.njk

```diff
@@ -9,6 +9,7 @@
 {% if '/features' in page.url %}
     <link rel="stylesheet" href="/_css/feature-support.css">
 {% endif %}
+    <link rel="icon" href="/favicon.png" sizes="32x32">
```

Comment:
size?

Reply:
the png is substantially smaller (1kb vs 15Kb) this offsets any avatar induced size increase.

Reply:
The current `favicon.ico` is 48×48px ([which was the smallest recommended web app manifest icon size](https://web.dev/learn/pwa/web-app-manifest)) and has a transfer size of 3,352 bytes. (gzip/Brotli works well on `ico`.)

<img width="661" alt="image" src="https://github.com/v8/v8.dev/assets/81942/675ac942-893d-4247-b65b-318f6f531f32">

I don’t see the PNG in this patch so I can’t compare, but from the HTML it looks like it’s 32×32px only?

Another option would be to do:

```html
<link rel="icon" href="/_img/v8-outline.svg">
```

This would add 0 extra image bytes to each page since we already use that SVG to display the logo in the header and footer. The downside is [it wouldn’t work in Safari](https://caniuse.com/link-icon-svg).

Reply:
Sticking with the png since it's smaller (1kb) and we have safari support.